db/seeds.rb

```diff
@@ -14,5 +14,6 @@
 # Create the default user account.
 user = UserEmail.find_by_email('test@example.org')
 unless user
-  User.create!(email: 'test@example.org', password: 'Coursemology!', role: :administrator)
+  User.create!(name: 'Administrator', email: 'test@example.org',
```

Comment:
No... that's too troublesome

On Sat, Jan 17, 2015 at 9:04 PM, Joel Low <notifications@github.com> wrote:

> In db/seeds.rb
> <https://github.com/Coursemology/coursemology2/pull/68#discussion-diff-23125123>
> :
>
> > @@ -14,5 +14,6 @@
> >  # Create the default user account.
> >  user = UserEmail.find_by_email('test@example.org')
> >  unless user
> > -  User.create!(email: 'test@example.org', password: 'Coursemology!', role: :administrator)
> > +  User.create!(name: 'Administrator', email: 'test@example.org',
>
> Well if we want to be consistent strictly speaking we should be placing
> one parameter per line... meh
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Coursemology/coursemology2/pull/68/files#r23125123>.
>